packages/shuttle_http/shuttle_http.0.11.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "Async library for HTTP/1.1 servers and clients"
+description:
+  "Shuttle_http is a low level library for implementing HTTP/1.1 web services and clients in OCaml."
+maintainer: "Anurag Soni <anurag@sonianurag.com>"
+authors: "Anurag Soni"
+license: "MIT"
+tags: ["http-server" "http-client" "http" "http1.1" "async"]
+homepage: "https://sr.ht/~soni/shuttle_http"
+bug-reports: "https://todo.sr.ht/~soni/shuttle_http"
+depends: [
+  "dune" {>= "3.1"}
+  "async" {>= "v0.16.0"}
+  "async_ssl" {>= "v0.16.0"}
```

Comment:
```suggestion
  "async_ssl" {>= "v0.16.0"}
  "async_websocket" {>= "v0.16.0"}
```

this seems to cause a lower bound failure if missing.

Reply:
Thank you! I’ll reproduce the changes in the upstream opam files. 

I’m usually good about setting lower bounds for packages, but I (incorrectly) assumed that since I define lower bounds for async and core async_websocket will automatically be locked to the version range >= 0.16 since it’s package definition specifies the lower bound on core and async to match >= 0.16 as well.

Reply:
Change reproduced upstream at https://git.sr.ht/~soni/shuttle_http/commit/b58bc34d23371f3d01ce6f9477f36aa61711b629